service/rpc/servicer.go

```diff
@@ -419,6 +419,98 @@ func (s *Servicer) BlockTransaction(ctx context.Context, request *types.BlockTra
 	}, nil
 }
 
+type CallMethod string
+
+const CeloCall CallMethod = "celo_call"
+const CeloGetLogs CallMethod = "celo_getLogs"
+
+func (cm CallMethod) String() string { return string(cm) }
+
+func AllCallMethods() []string {
+	return []string{CeloCall.String(), CeloGetLogs.String()}
+}
+
+func (s *Servicer) Call(ctx context.Context, request *types.CallRequest) (*types.CallResponse, *types.Error) {
+
+	switch request.Method {
+	case CeloCall.String():
+		var callArgs *airgap.CallArgs
+		if err := airgap.UnmarshallFromMap(request.Parameters, callArgs); err != nil {
```

Comment:
It would be nice to be able to pass in the `[contract].[methodName]` in the params, rather than passing the entire `CeloMethod` in the request as part of the CallArgs

Reply:
implemented in `txargs_marshaller.go`